### PR TITLE
SAR-393 Reduce logs spewed by device servers

### PR DESCRIPTION
**Pull request by kmadisa**
This change reduces the number of log entries made per attribute data parsed. I have change the log level of some log entries from `INFO` level to `DEBUG` level.
The rest of the other changes are just a result of applying the black formattor.

*Screenshots or code snippets (if appropriate):*
N/A

*Definition of Done Checklist*

- [x] Code meets our [python style guidelines](https://docs.google.com/document/d/1aZoIyR9tz5rCWr2qJKuMTmKp2IzHlFjrCFrpDDHFypM/edit?usp=sharing)?
- [x] Unit tested (coded, passed, included)?
- [x] Requested at least 2 reviewers?
- [x] Commented code, particularly in hard-to-understand areas?
- [x] Made corresponding changes to the documentation (e.g. Python documentation, System Engineering Documentation, version description updates, README file, etc)?

JIRA: [SAR-393](https://jira.skatelescope.org/browse/SAR-393)

### CODE REVIEW

File: tango_simlib/utilities/sim_xmi_parser.py [hunk 1]
```diff
@@ -649,12 +652,14 @@ def get_device_command_metadata(self):
...
-                        "The property '%s' cannot be translated to a "
-                        "corresponding parameter in the TANGO library",
-                        cmd_prop_name,
-                    )
+                    invalid_command_properties.add(cmd_prop_name)
+
             commands[cmd_name] = commands_metadata
+        MODULE_LOGGER.debug(
+            "The properties '%s' cannot be translated to "
+            "corresponding parameters in the TANGO library",
```
Comment (kmadisa) [inline tango_simlib/utilities/sim_xmi_parser.py:660]:
```suggestion
            "corresponding command parameters in the TANGO library",
```

File: tango_simlib/utilities/fandango_json_parser.py [hunk 2]
```diff
@@ -114,11 +115,13 @@ def preprocess_command_types(self, command_data):
...
-                        "parameter in the TANGO library",
-                        cmd_prop,
-                    )
+                    invalid_command_properties.add(cmd_prop)
+
+        MODULE_LOGGER.debug(
+            "The properties '%s' cannot be translated to a corresponding "
+            "parameters in the TANGO library",
+            invalid_command_properties,
+        )
```
Comment (samueltwum1) [inline tango_simlib/utilities/fandango_json_parser.py:124]: This is much better. That's where most of the noisy logs were coming from.

Should we not wrap this in an `if block` like on line 158?

**Review by samueltwum1 — APPROVED** [review]
LGTM. Can be merged if the pipeline passes.

File: tango_simlib/utilities/fandango_json_parser.py (hunk 2 above)
Comment (kmadisa) [reply tango_simlib/utilities/fandango_json_parser.py:124]: Thanks. I will do that.

**Comment by kmadisa** [comment]
Merging this PR as the Jenkins' build passed.
Screenshot from 2022-06-03 15-22-54

**Pull request merged**